panos/firewall.py

```diff
@@ -505,7 +505,19 @@ def set_shared_policy_synced(self, sync_status):
 
 
 class FirewallCommit(object):
-    """Normalization of a firewall commit."""
+    """Normalization of a firewall commit.
+
+    Instances of this class can be passed in to ``panos.firewall.Firewall.commit()``.
```

Comment:
Can you change each of these to a link? There are 3 of them, one for each class.

I believe you can do this by changing ``` ``panos.firewall.Firewall.commit()`` ``` to ``` :meth:`panos.firewall.Firewall.commit()` ```.

Reply:
Also mention how to pass it in, ie. the `cmd=` argument.